frontend/src/lib/components/leaderboard/leaderboard.svelte

```diff
@@ -104,7 +118,7 @@
                   </div>
                 {/if}
                 <span>
-                  {mmr ?? '🐣'}
```

Comment:
Why'd this change from static to string interpolation? 😅

Reply:
fixed